dipper/curie_map.yaml

```diff
@@ -1,3 +1,7 @@
+# Skolemize Blank Nodes
+# overwrite iri fragment with '' for unresovable bnodes
+'_':    'https://monarchinitiative.org/.well-known/genid/'
```

Comment:
@cmungall is there a way in owl to declare a node as anonymous?

The owlapi has this function: 
```
public interface OWLIndividual extends OWLObject, OWLPropertyAssertionObject {
...
    /**
     * Determines if this object is an instance of
     * {@link org.semanticweb.owlapi.model.OWLAnonymousIndividual} Note that
     * this method is the dual of {@link #isNamed()}.
     * 
     * @return {@code true} if this object represents an anonymous individual (
     *         {@code OWLAnonymousIndividual)} or {@code false} if this object
     *         represents a named individual ( {@code OWLIndividual})
     */
    boolean isAnonymous();
```